action.yml

```
@@ -95,6 +95,10 @@ inputs:
     description: "Directory to store output files."
     required: false
     default: ".github/outputs"
+  match_directories:
+    description: "Indicates whether to include match directories"
+    default: "true"
+    required: true
```

```suggestion
    required: false
```
probably shouldn't be required, right?

---

Yes, It should.